### PR TITLE
[ES] Fuzzy Bugs

### PR DESCRIPTION
**Pull request by luischos**

### CODE REVIEW

**Comment by qwrtln** [comment]
po4a check

File: translations/all_map_locations.tex/es.po [hunk 1]
```diff
@@ -369,38 +291,20 @@ msgid ""
...
-"\n"
 "  \\bigskip\n"
 "\n"
 "  \\phantom{j}\\textbf{Tumba del Guerrero}\\\\\n"
 "  \\framedimage[\\linewidth]{\\map_locations/warriors_tomb.jpg}\\\\\n"
 "\n"
 "  \\bigskip\n"
 "\n"
-"  \\phantom{j}\\textbf{Jardín Místico}\\\\\n"
+"  \\textbf{Jardín Místico}\\\\\n"
```
Comment (qwrtln) [inline translations/all_map_locations.tex/es.po:307]:
```suggestion
"  \\phantom{j}\\\\textbf{Jardín Místico}\\phantom{j}\\\\\\\n"
```
Return it, as it's breaking the layout. Look at the picture now - it's not aligned with the picture next to it.

**Comment by luischos** [comment]
Done, and all passed :D

**Comment by qwrtln** [comment]
Yesss, and just to show you the magic:

**Comment by qwrtln** [comment]
po4a commit

**Review by qwrtln — APPROVED** [review]

**Pull request merged**